packages/verifiable-credentials/src/credentials.ts

```diff
@@ -86,20 +84,22 @@ export default class Credentials {
 	 * Verify a Verifiable Presentation DID-JWT
 	 *
 	 * @param {string} vpJwt
+	 * @param {string} didRegistryEndpoint
 	 */
-	async verifyPresentation(vpJwt: string): Promise<unknown> {
-		const resolver = this.getResolver();
+	static async verifyPresentation(vpJwt: string, didRegistryEndpoint: string): Promise<any> {
+		const resolver = Credentials.getResolver(didRegistryEndpoint);
 		return verifyPresentation(vpJwt, resolver);
 	}
 
 	/**
 	 * Verify a Verifiable Credential DID-JWT
 	 *
 	 * @param {string} vcJwt
+	 * @param {string} didRegistryEndpoint
 	 * @param {string} currentDateTime to allow the client to migrate cases where the datetime is incorrect on the local computer
 	 */
-	async verifyCredential(vcJwt: string, currentDateTime?: string): Promise<any> {
-		const resolver = this.getResolver();
+	async verifyCredential(vcJwt: string, didRegistryEndpoint: string, currentDateTime?: string): Promise<any> {
```

Comment:
Will need to pass in a third variable `Verida.EnvironemntType`.

Reply:
I have also passed the `EnvironemntType` as well